src/hardware/vga_other.cpp

```diff
@@ -559,9 +566,16 @@ static void update_cga16_color_pcjr()
 			const auto G = normalize_and_apply_gamma(Y - 0.2721f * I - 0.6474f * Q);
 			const auto B = normalize_and_apply_gamma(Y - 1.1069f * I + 1.7046f * Q);
 
-			auto to_8bit_without_gamma = [=](float v) -> uint8_t {
-				const auto without_gamma = clamp(powf(v, 1 / gamma), 0.0f, 1.0f);
-				return static_cast<uint8_t>(255 * without_gamma);
+			auto to_8bit_without_gamma = [=](const float v) -> uint8_t {
+				// Only operate on reasonably positive v-values
+				if (!isnormal(v) || v <= 0.0f)
+					return 0;
+
+				// Remove gamma and scale the result to the 8-bit range
+				constexpr int max_8bit = UINT8_MAX;
+				const auto no_gamma = powf(v, 1.0f / gamma) * max_8bit;
```

Comment:
picky: maybe instead of "removing the gamma" you could use the terms "gamma decoding" or "linear RGB".

Reply:
Thanks @johnnovak ; also renamed the lamda function to `to_linear_rgb`.